src/Colors.css

```diff
@@ -4,5 +4,6 @@
   --color-accent: #009fcc;
   --color-accent-onAction: #0087ac;
   --color-text-onDark: white;
+  --color-text-onDark-dim: #ffffffcf;
```

Comment:
isnt the cf at the end 2 letters too much? i thought hex code are always 6 digits/letters

Reply:
I agree that looks a bit strange ?

Reply:
The last ones are for opacity. It makes the color a bit translucent. :)